openmetadata-service/src/main/java/org/openmetadata/service/security/JwtFilter.java

```diff
@@ -90,7 +90,9 @@ public class JwtFilter implements ContainerRequestFilter {
           "v1/users/password/reset",
           "v1/users/checkEmailInUse",
           "v1/users/login",
-          "v1/users/refresh");
+          "v1/users/refresh",
+          "v1/collate/apps/slack/callback",
```

Comment:
this cannot be here , we need to think of other solution here

Reply:
and don't add /collate there

Reply:
I want to expose those endpoints. I am validating them internally using Bolt. I will check of other way. Noted.

Reply:
Updated.